### DISCUSSION TITLE
Pimcore doesnot  update the Pimcore object (item), when   import  from  spreadsheet

### DISCUSSION THREAD

**Discussion by TestingKbiz** [pimcore/pimcore#8625] [Q&A / Support]
I have tried to update the  Pimcore object from the CSV file but it does not update the object.
here is my screenshot before import: https://prnt.sc/110r47d
tried import from here:   https://prnt.sc/110r7j4

I have also tried to import  CSV,  Xls, etc but no luck with it.

please help me with how to import  CSV or Xls to update the item.
Also I have  recorded a video while  importing : https://www.loom.com/share/d905e3e2198448f09fbf0d26d4ce3a24
here is the file https://docs.google.com/spreadsheets/d/1XPs-9s3OYzSzA-Ovf4QcxCoTDPGx6R43MCy2TlfWot0/edit#gid=276330350

**Comment by dvesh3** [comment]
@TestingKbiz you are using class import feature to import data object, which is wrong. please right click on your object  -> CSV Import -> ClassType and upload spreadsheet.
